packages/cashier/src/Containers/crypto-deposit.jsx

```diff
@@ -0,0 +1,118 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Button, ButtonLink, Clipboard, Loading, Text, Icon } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+import { CryptoConfig, isMobile } from '@deriv/shared';
+import QRCode from 'qrcode.react';
+import { connect } from 'Stores/connect';
+import '../Sass/deposit.scss';
+
+const CryptoDeposit = ({
+    api_error,
+    currency,
+    deposit_address,
+    is_deposit_address_loading,
+    pollApiForDepositAddress,
+}) => {
+    React.useEffect(() => {
+        return () => pollApiForDepositAddress(false);
+    }, [pollApiForDepositAddress]);
+
+    if (is_deposit_address_loading) {
+        return <Loading is_fullscreen={false} />;
+    }
+
+    const currency_name = CryptoConfig.get()[currency].name;
+
+    return (
+        <div className='cashier__wrapper crypto-deposit__wrapper'>
+            <div className='crypto-deposit__transaction-wrapper'>
+                <Icon icon={`IcCurrency${currency}`} size={64} />
+                <Text
+                    className='crypto-deposit__transaction-currency'
+                    weight='bold'
+                    as='p'
+                    line_height='m'
+                    size={isMobile() ? 'xs' : 's'}
+                    align='center'
+                >
+                    <Localize
+                        i18n_default_text='Send only {{currency}} ({{currency_symbol}}) to this address.'
+                        values={{
+                            currency: currency_name,
+                            currency_symbol: currency?.toUpperCase(),
+                        }}
+                    />
+                </Text>
+                <Text as='p' line_height='m' size={isMobile() ? 'xs' : 's'} align='center'>
+                    <Localize i18n_default_text="Do not send any other currency to the following address. Otherwise, you'll lose funds." />
+                </Text>
+
+                {api_error ? (
+                    <div className='crypto-api-error'>
+                        <Text as='p' align='center' size='xs' className='crypto-api-error__text'>
+                            <Icon icon='IcAlertWarning' />
+                            <Localize i18n_default_text='Our server cannot retrieve an address' />
+                        </Text>
+                        <Button text={localize('Refresh')} onClick={() => window.location.reload()} secondary small />
```

Comment:
I don't think clicking on `Refresh` button should reload the entire window. It should only call the API for deposit address.